docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -241,41 +241,59 @@ Step 2: Send **at least 1 Holesky ETH** to the “address” field referenced in
 
 #### Configuration Setup
 
-You can create the config files needed for operator registration using the below command:
+**Step 1:** Create the config files needed for operator registration using the below command:
 
 ```
 eigenlayer operator config create
 ```
 
 When prompted for operator address, make sure your operator address is same as the ecdsa key address you created/imported in key creation steps. 
 
-It will create two files: `operator.yaml` and `metadata.json` After filling the details in `metadata.json`, please upload this into a publicly accessible location and fill that url in `operator.yaml`. A valid metadata url is required for successful registration. An example operator.yaml file is provided for your reference here: [operator.yaml](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/pkg/operator/config/operator-config-example.yaml) .
+The command will create two files: `operator.yaml` and `metadata.json`.
+
+**Step 2:** Modify `metadata.json`, upload and ensure it is publicly accessible:
```

Comment:
For mainnet - we are planning to enforce this public URL to be hosted on github due to some security recommendations. we should maybe add it? not sure how to make it explicit that this requirement is only for github

Reply:
@Madhur can you confirm this decision is finalized? "public URL to be hosted on github" for both metadata.json and logo png file?

If the decision is final, I agree we should absolutely communicate that *prominently*

Reply:
Doc updated to reflect "github.com" hosting is mandatory for Mainnet Operators.